tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -213,8 +209,7 @@ class MklConvFwdPrimitive : public MklPrimitive {
           filter_md(nullptr),
           bias_md(nullptr),
           fwd_pd(nullptr),
-          conv_fwd(nullptr),
-          fwd_stream(nullptr) {
+          conv_fwd(nullptr) {
```

Comment:
we are planning to move engine and primitive to the base class since they are common features of all primitives. I have already moved the engine. Moving stream is slightly more complex. Besides, with the DNNL threadpool , stream creation needs OpKernelContext and passing this through primitive cache is cumbersome.